testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/SocialLoginTest.java

```diff
@@ -552,6 +552,7 @@ private void doLogin() {
             log.infof("current URL: %s", driver.getCurrentUrl());
             log.infof("performing log in to '%s' ...", currentTestProvider.id());
             currentSocialLoginPage.login(getConfig("username"), getConfig("password"));
+            DroneUtils.getCurrentDriver().findElement(By.xpath("//button//span[contains(.,'Continue')]")).click();
```

Comment:
This will probably affect all provider tests. Please confirm if it's working properly.